pkg/porter/cnab_test.go

```diff
@@ -98,6 +101,13 @@ func TestSharedOptions_validateBundleJson(t *testing.T) {
 }
 
 func Test_bundleFileOptions(t *testing.T) {
+	absWantFile, err := filepath.Abs(filepath.FromSlash("/" + config.Name))
```

Comment:
there is some funny handling here I believe, depending on how the it is seen as relative. I don't see anything wrong but double check you are getting the expected values in the test.

Reply:
thanks, I believe it's correct